tests/integration/targets/dcnm_vrf/tests/dcnm/merged.yaml

```diff
@@ -115,6 +115,107 @@
     - 'result.changed == false'
     - 'result.response|length == 0'
 
+- name: MERGED - Clean up any existing vrfs
+  cisco.dcnm.dcnm_vrf:
+    fabric: "{{ ansible_it_fabric }}"
+    state: deleted
+
+- name: MERGED - sleep for 10 seconds for DCNM to completely update the state
+  wait_for:
+    timeout: 10
+
+- name: MERGED - Create, Attach and Deploy new VRF - VLAN/VRF LITE EXTENSION Provided by the User in one switch
```

Comment:
I tried to run these VRF LITE tests but was not clear on what the topology should look like.  Can you share the topology you were using to run these tests?

Also, it looks like `Ethernet1/16` was significant for your topology but we need to think about making that interface a variable so that it can be passed in just like the switch ip and fabric name.

Reply:
The topology is nothing but 2 fabric, 1 external fabric and 1 normal fabric, 2 back to back connected devices each in one of the fabrics. E1/16 is not that significant in our code as that is the connected interface between these 2 devices in the topology and it is detected automatically by the dcnm. The interface is not at all mandatory for VRF LITE as we will pick that info from the dcnm only if not specified in the playbook.

Reply:
Understood, but when we run these tests it could be a different interface name that connects the fabrics right?  Just want to make sure we can pass whatever int is needed to the test instead of hard coding it.

Reply:
ok will change the interface details to variable.

Reply:
done the same in IT cases.